plugins/jigoshop-web-optimization-system/src/Jigoshop/Web/Optimization.php

```diff
@@ -220,7 +220,7 @@ private function get_current_page()
 
 	private function get_source_path($src)
 	{
-		return preg_replace('@'.str_replace('http', 'http(s)?', WP_CONTENT_URL).'@', WP_CONTENT_DIR, $src);
+		return preg_replace('@'.str_replace(array('http://', 'https://'), '', WP_CONTENT_URL).'@', WP_CONTENT_DIR, str_replace(array('http://', 'https://'), '',$src));
```

Comment:
The same effect: `return str_replace(str_replace(array('http://', 'https://'), '', WP_CONTENT_URL), WP_CONTENT_DIR, str_replace(array('http://', 'https://'), '', $src);` with much less calculation consumption.

When you are not using anything related to regular expressions - do not use `preg_` or (even worse) `ereg_` functions.